codegen/core/src/main/scala/kalix/codegen/ModelBuilder.scala

```diff
@@ -709,21 +710,23 @@ object ModelBuilder {
       State(resolveMessageType(workflowDef.getState, protoPackageName, additionalDescriptors)))
   }
 
+  @nowarn("msg=deprecated")
```

Comment:
I'm confused. Calling `getEntityType` gives a deprecation and therefore we are excluding with `@nowarn`? 

Why we are doing this?

Reply:
The `@nowarn` to allow compilation.

Reply:
Got it now.. had to look at https://github.com/lightbend/kalix-jvm-sdk/pull/2260/files#diff-a2f65419f287e3a9648619b8f1f6b70e1da8169481f66d2aae60ccb0b84c43fbR793 to understand it better.

Btw, same fix was applied previously [here](https://github.com/lightbend/kalix-jvm-sdk/pull/2167) but we missed these 2 cases I guess.